python_modules/libraries/dagster-powerbi/dagster_powerbi/translator.py

```diff
@@ -132,11 +133,20 @@ def get_dashboard_spec(self, data: PowerBIContentData) -> AssetSpec:
             self.get_report_asset_key(self.workspace_data.reports_by_id[report_id])
             for report_id in tile_report_ids
         ]
+        url = data.properties.get("webUrl")
+        metadata = (
+            {
+                "url": MetadataValue.url(url),
```

Comment:
nitpick, I think this could fit on fewer lines without the comma?

Reply:
I wonder if the URL should have some kind of prefix to disambiguate it from other URLs we might want to share for the same asset? Or is there some chance we should reuse `dagster/uri` for this? Would we want to give it the same visual treatment as an S3 URI?

Reply:
I could go either way on this - updated to `web_url` for now, but could see this going in `dagster/uri` or some sort of `dagster/view_url` canonical/promoted metadata key.

I do think it feels a little different from `dagster/uri`, which in my mind at least I conceptualize as more as an address than just a plain URL.

Reply:
Thoughts on `dagster_powerbi/web_url`? I generally think we should always be using a prefix when yielding metadata from a Dagster library, so that it's clear who's taking responsibility for defining the meaning of that entry.

Reply:
Updated to use `dagster_powerbi/web_url`